docs/topics/time.md

```diff
@@ -0,0 +1,373 @@
+
+You can also perform basic arithmetic with `Durations`:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.seconds
+
+fun main() {
+    //sampleStart
+    val fiveSeconds: Duration = 5.seconds
+    val thirtySeconds: Duration = 30.seconds
+
+    println(fiveSeconds + thirtySeconds)
+    // 35s
+    println(thirtySeconds -  fiveSeconds)
+    // 25s
+    println(fiveSeconds * 2)
+    // 10s
+    println(thirtySeconds / 2)
+    // 15s
+    println(thirtySeconds / fiveSeconds)
+    // 6.0
+    println(-thirtySeconds)
+    // -30s
+    println((-thirtySeconds).absoluteValue)
+    // 30s
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+### Print duration
+
+To print a `Duration`, you have already seen that you can use `println()` and `print()` functions. By default, the
+time is reported using each unit that is present. For example: `1h 0m 45.677s`.
+
+To configure the output, use the `.toString()` function with your desired `Duration.Unit` and number of decimal places
+as function parameters:
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.DurationUnit
+
+fun main() {
+    //sampleStart
+    val fiveSeconds: Duration = 5.seconds
+    
+    // Print in minutes with 2 decimal places
+    println(fiveSeconds.toString(DurationUnit.MINUTES, 2))
+    // 0,08m
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+To print the `Duration` as an [ISO-8601-compatible](https://en.wikipedia.org/wiki/ISO_8601) string, use the [`toIsoString()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/to-iso-string.html)
+function:
+
+```kotlin
+import kotlin.time.Duration.Companion.seconds
+
+fun main() {
+    //sampleStart
+    println(86420.seconds.toIsoString()) // PT24H0M20S
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+### Convert duration
+
+To express your `Duration` in a different `Duration.Unit` you can use the properties:
+* `inWholeNanoseconds`
+* `inWholeMicroseconds`
+* `inWholeSeconds`
+* `inWholeMinutes`
+* `inWholeHours`
+* `inWholeDays`
+
+For example:
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
+
+fun main() {
+    //sampleStart
+    val thirtyMinutes: Duration = 30.minutes
+    println(thirtyMinutes.inWholeSeconds)
+    // 1800
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+Alternatively, you can use your desired `Duration.Unit` as a function parameter in the following extension functions:
+* `toInt()`
+* `toDouble()`
+* `toLong()`
+
+For example:
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.DurationUnit
+
+fun main() {
+    //sampleStart
+    val thirtyMinutes: Duration = 30.minutes
+    println(thirtyMinutes.toDouble(DurationUnit.SECONDS))
```

Comment:
For converting to double better to use a value that actually produces a fractional part.